apps/i18n/common/en_us.json

```diff
@@ -784,6 +785,7 @@
   "exampleErrorMessage": "The function {functionName} has one or more examples that need adjusting. Make sure they match your definition and answer the question.",
   "examplesFailedOnClose": "One or more of your examples do not match your definition. Check your examples before closing",
   "exampleSolution": "Example Solution {number}",
+  "explainTtsAutoplay": "For Chrome, Internet Explorer, and Edge browsers only: When Automatic Instruction Playback is on, the instructions will be automatically read when a student opens a level. This gives students still learning to read or with difficulty reading the opportunity to participate in the lessons.",
```

Comment:
(minor) The grammar around "with difficulty reading" is a little awkward.  Could you double-check with Amanda on the string here?  My suggestion would be something like: "... This makes it easier for students who are not yet fluent readers to complete the lessons."

Reply:
This text was provided by Amanda, I'll check with her again!

Reply:
Updated!